include/mgba/internal/arm/decoder.h

```diff
@@ -196,6 +196,10 @@ enum ARMMnemonic {
 	ARM_MN_SMULBT,
 	ARM_MN_SMULTB,
 	ARM_MN_SMULTT,
+	ARM_MN_SMLAWB,
```

Comment:
This table should be in alphabetical order. The fact that it's slightly out of order right now is an oversight, and it would be appreciated if you could correct that while you're here.